crates/core/src/server/http_gateway.rs

```diff
@@ -29,22 +30,33 @@ impl std::ops::Deref for HttpGatewayRequest {
     }
 }
 
+pub type AttestedContractMap = Arc<RwLock<HashMap<AuthToken, (ContractInstanceId, ClientId)>>>;
```

Comment:
You can use Arc<DashMap<_, _>> since we are using it in other places, should be a bit more performant, instead, this one I didn't see.

Reply:
I did consider that but thought it wouldn't make a difference in practice given how it's used.